packages/components/modules/payments/web/InvoiceListTable/components/InvoiceItemWrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import { DATE_FORMAT, formatDateFromApi } from '@baseapp-frontend/utils'
+
+import { InvoiceItemWrapperProps } from '../types'
+import InvoiceItemTableRow from './InvoiceItemTableRow'
+import MobileInvoiceItemTableRow from './MobileInvoiceItemTableRow'
+
+const InvoiceItemWrapper = ({ row, rowProps, cellProps, smDown }: InvoiceItemWrapperProps) => {
+  const amountDue = (row.amountDue / 100).toFixed(2) ?? ''
+  const formattedDate =
+    formatDateFromApi(row.webhooksDeliveredAt, { toFormat: DATE_FORMAT[1] }) ?? ''
+
+  const getStatusColor = (status: string) => {
+    if (status === 'paid') return 'success'
+    if (status === 'failed') return 'error'
+    return 'warning'
+  }
+
+  return smDown ? (
```

Comment:
would be nice to use if/return instead of ternary, just in case we have a third component to return (loading state, etc...)